index.js

```diff
@@ -138,7 +139,7 @@ class ReactDocGenMarkdownRenderer {
   }
 
   render(file, docs, composes) {
-    const componentName = path.basename(file, path.extname(file));
+    const componentName = this.options.componentsName === 'fileName' ? path.basename(file, path.extname(file)) : docs.displayName;
```

Comment:
I was kinda thinking to always use `docs.displayName` and if it doesn't exist just fallback to `path.basename(file, path.extname(file))`.

Something like this:
```js
const componentName = docs.displayName ? docs.displayName : path.basename(file, path.extname(file));
```

That way we don't need the extra configuration and it's more dynamic, some components can have `displayName` and some don't it doesn't have to be a cross-cutting concern.

WDYT?

Reply:
LGTM 👌 

I was thinking you wanted to add the option to choose between the two behaviors. But always use `docs.displayName` seems to be a better option.